### PR TITLE
Add support for 'Colortemperaturepicker' widget

### PR DESCRIPTION
**Pull request by maniac103**

### CODE REVIEW

**Comment by mueller-ma** [comment]
I tested it with my Nanoleafs and it works fine 👍

File: mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.kt [hunk 1]
```diff
@@ -1702,6 +1708,29 @@ class WidgetAdapter(
...
+        R.layout.widgetlist_colortemperatureitem,
+        R.layout.widgetlist_colortemperatureitem_compact
+    ) {
+        private val previewImage = itemView.findViewById<ImageView>(R.id.current_temperature)
+
+        override fun bind(widget: Widget) {
+            super.bind(widget)
+            val drawable = (widget.state ?: widget.item?.state)?.asNumber?.let { state ->
+                // if not Kelvin, assume Mirek
+                val kelvin = if (state.unit == "K") state.value else 1000000F / state.value
```
Comment (maniac103) [inline mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.kt:1722]: Alternative logic mentioned [here](https://github.com/openhab/openhab-core/issues/3891#issuecomment-2509668785) ... not sure what I prefer yet.

File: mobile/src/main/java/org/openhab/habdroid/ui/WidgetDetailBottomSheets.kt [hunk 2]
```diff
@@ -98,6 +102,51 @@ class SliderBottomSheet : AbstractWidgetBottomSheet(), WidgetSlider.UpdateListen
...
+        connection?.httpClient?.sendItemUpdate(item, state)
+    }
+
+    override fun onLayoutChange(view: View, l: Int, t: Int, r: Int, b: Int, ol: Int, ot: Int, or: Int, ob: Int) {
+        applyColorTemperatureGradientToTrack(view as WidgetSlider, r - l, b - t)
+    }
+
+    private fun applyColorTemperatureGradientToTrack(slider: WidgetSlider, width: Int, height: Int) {
+        val min = widget.minValue
+        val max = widget.maxValue
```
Comment (maniac103) [inline mobile/src/main/java/org/openhab/habdroid/ui/WidgetDetailBottomSheets.kt:128]: This currently assumes Kelvin ... translation needed (as mentioned above)?

File: mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.kt (hunk 1 above)
Comment (maniac103) [reply mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.kt]: I've implemented the distinction by value (< 1000 is Mirek / > 1000 is Kelvin) for now.

**Pull request merged**